src/EditorFeatures/CSharpTest/UseCompoundAssignment/UseCompoundAssignmentTests.cs

```diff
@@ -215,6 +216,40 @@ void M(int a)
 }");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseCompoundAssignment)]
+        public async Task TestCoalesceExpressionCSharp8OrGreater()
+        {
+            await TestInRegularAndScriptAsync(
+@"public class C
+{
+    void M(int? a)
+    {
+        a [||]= a ?? 10;
+    }
+}",
+@"public class C
+{
+    void M(int? a)
+    {
+        a ??= 10;
+    }
+}", parseOptions: new CSharpParseOptions(LanguageVersion.CSharp8));
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseCompoundAssignment)]
+        public async Task TestCoalesceExpressionCSharp7()
+        {
+            await TestMissingAsync(
+@"public class C
+{
+    void M(int? a)
+    {
+        a [||]= a ?? 10;
+    }
+}",
+    new TestParameters(parseOptions: new CSharpParseOptions(LanguageVersion.CSharp7)));
```

Comment:
nit: consider changing this to C# 7.3 (to better pin down that this is enabled from 8.0 onward)

Reply:
I thought about it but didn't think it mattered much, updated now.